app/checkout/[userId]/page.tsx

```diff
@@ -84,16 +103,28 @@ export default function CheckOut({ params }: CheckOutProps) {
             <VisitInfoForm
                 initialVisitData={visitsData[0]}
                 onSubmit={() => setShow(true)}
+                onChange={onChange}
```

Comment:
Nit: we can pass `setVisitFormData` here

Reply:
Done!